### ISSUE TITLE
Accessibility - default hyperlink styles have no :focus style!

### ISSUE THREAD

**Issue by Stephen2** [ant-design/ant-design#22503]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Improves accessibility

### What does the proposed API look like?
```
a:focus { the same styles as a:hover };
```

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/5597 was removed for some reason. I will add a less variable `@link-focus-decoration` to allow customization.

**Comment by yoyo837** [comment]
ref: #15759

**Comment by Stephen2** [comment]
I disagree strongly with closing this issue and doing nothing by default.  All interactive (TABable) elements need a visual indicator of focus.

Having our default as a critical FAIL on accessibility is a very poor decision and moves us backwards, not forwards.

If you don't want it to be underline - at LEAST make it the subtle opacity that `:hover` gets by default.  (This is so subtle in fact, that it probably is borderline to pass, and I think `:focus` should have a more striking visual effect so the poorly sighted users know where their focus is...)

**Comment by afc163** [comment]
#22511
